packages/blocks/src/_presets/index.ts

```diff
@@ -190,7 +190,7 @@ export const PagePreset: BlockSpec[] = [
       },
     },
   },
-  embedGithubBlockSpec,
+  EmbedGithubBlockSpec,
 ];
 
 export const EdgelessPreset: BlockSpec[] = [
```

Comment:
We may also want to rename `_presets` in blocks to `_blocks`

Reply:
How about make it `_specs`?